workloads/kube-burner/common.sh

```diff
@@ -177,3 +177,122 @@ remove_update_taint() {
   # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
   oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
 }
+
+
+label_node_label() {
+  colon_param=$(echo $1 | tr "=" ":" | sed 's/:/: /g')
```

Comment:
Is this colon_param variable actually used?

Reply:
No it currently is not but I was trying to work towards combining this function and label_nodes to be more generic to take in the wanted label and export the correct parameters with a colon or with equals. Will update if I finalize it